hazelcast/serialization/api.py

```diff
@@ -2593,3 +2594,57 @@ def get_type_name(self) -> str:
         Returns:
             The type name.
         """
+
+
+class FieldKind(enum.IntEnum):
+    """
+    Represents the types of the fields used in the Compact serialization.
+    """
+
+    NOT_AVAILABLE = 0
+    """
+    Represents fields that do not exist.
+    """
+
+    BOOLEAN = 1
+    ARRAY_OF_BOOLEAN = 2
+    INT8 = 3
+    ARRAY_OF_INT8 = 4
+    INT16 = 7
+    ARRAY_OF_INT16 = 8
+    INT32 = 9
+    ARRAY_OF_INT32 = 10
+    INT64 = 11
+    ARRAY_OF_INT64 = 12
+    FLOAT32 = 13
+    ARRAY_OF_FLOAT32 = 14
+    FLOAT64 = 15
+    ARRAY_OF_FLOAT64 = 16
+    STRING = 17
+    ARRAY_OF_STRING = 18
+    DECIMAL = 19
+    ARRAY_OF_DECIMAL = 20
+    TIME = 21
+    ARRAY_OF_TIME = 22
+    DATE = 23
+    ARRAY_OF_DATE = 24
+    TIMESTAMP = 25
+    ARRAY_OF_TIMESTAMP = 26
+    TIMESTAMP_WITH_TIMEZONE = 27
+    ARRAY_OF_TIMESTAMP_WITH_TIMEZONE = 28
+    COMPACT = 29
+    ARRAY_OF_COMPACT = 30
+    NULLABLE_BOOLEAN = 33
```

Comment:
Adding unsupported fields and 31, 32 may be nice as documentation.

Reply:
OK, let me do it. I will also do the same for char and array of char